app/views/vacancies/show.html.haml

```diff
@@ -90,7 +90,7 @@
         %dt= t('vacancies.working_pattern')
         %dd= @vacancy.working_pattern
 
-        - if @vacancy.part_time?
+        - if @vacancy.part_time? && @vacancy.weekly_hours
```

Comment:
I'd have expected this to break a feature test, which might highlight the absence of any test coverage on this behaviour.

Reply:
I had a crack in users_can_view_vacancies_spec.rb
```
context 'when the vacancy is part_time' do
    scenario 'Shows the weekly hours if there are weekly_hours' do
      vacancy = create(:vacancy, working_pattern: :part_time, weekly_hours: '5')
      Vacancy.__elasticsearch__.client.indices.flush
      visit vacancies_path
      expect(page).to have_content(vacancy.weekly_hours)
    end

    scenario 'Does not show if there are not the weekly_hours' do
      vacancy = create(:vacancy, working_pattern: :part_time, weekly_hours: nil)
      Vacancy.__elasticsearch__.client.indices.flush
      visit vacancies_path
      expect(page).to have_content(vacancy.weekly_hours)
    end
  end
```